tcms/testruns/models.py

```diff
@@ -157,9 +156,8 @@ def add_case_run(self, case, case_run_status=1, assignee=None,
             or (self.default_tester_id and self.default_tester)
 
         get_caserun_status = TestCaseRunStatus.objects.get
-        _case_run_status = isinstance(case_run_status, int) \
-            and get_caserun_status(id=case_run_status) \
-            or case_run_status
+        _case_run_status = get_caserun_status(id=case_run_status) \
```

Comment:
As far as I can see `get_caserun_status` is not used anywhere else. Just use the get method directly here.